src/wire/number.utils.ts

```diff
@@ -60,7 +60,7 @@ export const floatToBuf = (v: number) => {
 }
 
 export const doubleToBuf = (v: number) => {
-  const b = Buffer.allocUnsafe(4);
+  const b = Buffer.allocUnsafe(8);
```

Comment:
What's the reason of this modification?

Reply:
it was a mistake and a bug ! this would throw next line when attempting to write the 8 bit from a Double. but it's only used by Message.Header with type double and it wasn't used by me or test covered before this PR it was below radar :sweat_smile: